Library/Formula/mg3a.rb

```diff
@@ -0,0 +1,39 @@
+class Mg3a < Formula
+  homepage "http://www.bengtl.net/files/mg3a/"
```

Comment:
Could you add a description (`desc`) above the homepage? That will also trigger a new build so we can look at those test failures again.